src/robusta/core/playbooks/container_playbook_utils.py

```diff
@@ -20,7 +20,13 @@ def create_container_graph(params: ResourceGraphEnricherParams, pod: Pod, contai
         if params.resource_type == "Memory" and limits > 0:
             memory_limit_in_bytes = limits * 1024 * 1024
             limit_line = XAxisLine(label="Memory Limit", value=memory_limit_in_bytes)
-            limit_lines = [limit_line]
+            limit_lines.append(limit_line)
```

Comment:
this seems to be missing the CPU limits + requests here
I would refactor this to a function that does all of this logic and returns the lines[] of the limits and call it here and in pod_graph_enricher

Reply:
done